test/demo.html

```diff
@@ -59,6 +59,18 @@
 
 <p>This button runs the JavaScript code written on it to MathQuill-ify the following <code>&lt;span&gt;</code> element into an editable math textbox: <button onclick="$(this).text('MQ(this.nextSibling).revert()'); MQ.MathField(this.nextSibling); var orig = arguments.callee; this.onclick = function(){ $(this).text('MQ.MathField(this.nextSibling)'); MQ(this.nextSibling).revert(); this.onclick = orig; };">MQ.MathField(this.nextSibling)</button><span>\frac{d}{dx}\sqrt{x} = \frac{d}{dx}x^{\frac{1}{2}} = \frac{1}{2}x^{-\frac{1}{2}} = \frac{1}{2\sqrt{x}}</span></p>
 
+<p>
+  Insert <button class="insert-trigger" data-latex="\sqrt{4}"><span class="mathquill-inert-math">\sqrt{4}</span></button> inside formulas below:<br>
+  <span class="mathquill-math-field"></span><br>
+  or here: <span class="mathquill-math-field">\frac{2}{3}</span>
+</p>
+
+<p>
+  But if you won't use InertMath, the result will be far from perfect:<br>
+  Insert <button class="insert-trigger" data-latex="\sqrt{4}"><span class="mathquill-math-field">\sqrt{4}</span></button> inside formulas below:<br>

```

Comment:
Removed this - was rubbish indeed. I edited previous example to make it more meaningful.